.test-infra/jenkins/CommonJobProperties.groovy

```diff
@@ -97,6 +97,7 @@ class CommonJobProperties {
         string("COVERALLS_REPO_TOKEN", "beam-coveralls-token")
       }
       timestamps()
+      colorizeOutput()
```

Comment:
For my education, where is this defined?

Reply:
It's from the ansicolor plugin, which I recently asked Infra to install.
Technically, the Jenkins DSL plugin defines `colorizeOutput`.